test_sb/sb.vp

```diff
@@ -0,0 +1,266 @@
+//; use POSIX;
```

Comment:
can we move this file to `sb/genesis/` directory? All genesis (`.vp`) files should go there.